[Dev] Update brace-expansion from 1.1.11 -> 1.1.12 and 2.0.1 -> 2.0.2

Tempo-anon opened the pull request:
## What are the changes the user will see?

None

## Why am I making these changes?

Dependabot vulnerability: https://github.com/Despair-Games/poketernity/security/dependabot/13

## What are the changes from a developer perspective?

Updated version numbers and hashes

## Screenshots/Videos

n/a

## How to test the changes?

n/a

## Checklist

- [ ] ⚠️ If this is a PR for `main` (such as a hotfix), has the game version been updated (`pnpm update-version:patch` / `pnpm update-version:minor`?
- [x] Otherwise: **I'm using `beta` as my base branch**
- [x] There is no overlap with another PR?
- [x] The PR is self-contained and cannot be split into smaller PRs?
- [x] Have I provided a clear explanation of the changes?
- [ ] Have I tested the changes manually?
<!-- We have heavily optimized our test suite, so please actually run the tests :) -->
- [ ] Are all unit tests still passing? (`pnpm test:silent`)
  - [ ] Have I created new automated tests (`pnpm test:create`) or updated existing tests related to the PR's changes?
- [ ] Have I provided screenshots/videos of the changes (if applicable)?
  - [ ] Have I made sure that any UI change works for both UI themes (dark and light)?

#### Are there any localization additions or changes? If so:

- [ ] Has a locales PR been created on the [locales](https://github.com/despair-games/poketernity-locales) repo?
  - [ ] If so, please leave a link to it here:
- [ ] Have I added the `Localization` tag to this PR?
<!-- not relevant for now - [ ] Has the translation team been contacted for proofreading/translation? -->
<!-- You can find a summarized version of the merging process surrounding locale PRs in your locale PR itself. For full instructions, check [localization.md](https://github.com/Despair-Games/poketernity/blob/beta/docs/localization.md) -->

#### If there are no locale changes:
- [ ] Have I made sure **not** to commit any changes to the locale repo on this branch?
<!-- check the `Files Changed` tab on the PR to be sure. 
`public/locales` should not appear here, or it will create needless conflicts for future PRs and could potentially roll back changes already merged to beta. -->

<!-- How to fix it if no: -->
<!-- #### Using the Command Line:
- Go to https://github.com/Despair-Games/poketernity/tree/beta/public and copy the hash of the current locale commit beta is pointing to
- If the hash corresponds to the latest commit in the locale repo:
  - `pnpm update-locales:remote`
  - `git add public/locales`
  - make your commit, push etc
- If it's not the latest commit:
  - `git checkout beta`
  - if not up to date: `git pull`. Otherwise: `pnpm update-locales:remote`
  - `git checkout {this pr's branch}`
  - `git add public/locales`
  - make your commit, push etc

 /!\ anyone using another tool, feel free to add instructions there /!\

You may have to do this again later and fix conflicts if beta keeps updating the locale repo.
**When fixing conflicts, make sure you prioritize the latest commit between the one on beta and this branch, to avoid any rollbacks.** -->

DayKev commented:
That's weird, you shouldn't need to install the dependency in `package.json`. How did you update it in `pnpm-lock.yaml` initially? When I ran `pnpm update brace-expansion` it updated 7 different entries, but you only have 2 updated.

Tempo-anon commented:
Initially I just changed the version numbers and updated the hash. Since that didn't work I did `pnpm i brace-expansion@2.0.2` and `pnpm i brace-expansion@1.1.12`

DayKev commented:
It seems like if you reset to the latest beta commit and then run `pnpm update brace-expansion` and commit that, it should update properly without requiring us to add it as a dependency.

Tempo-anon committed:
try kev suggestion
